src/main/java/core/basesyntax/service/storageservice/StorageReportService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service.storageservice;
+
+public interface StorageReportService {
+    String getReport();
+}
```

Comment:
can we move this method to StorageService?

Reply:
done